fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/TabDrawerRobot.kt

```diff
@@ -297,12 +296,6 @@ class TabDrawerRobot {
             itemContainingText(getStringResource(R.string.synced_tabs_sign_in_message)),
             itemContainingText(getStringResource(R.string.sync_sign_in)),
         )
-        assertItemWithResIdAndDescriptionExists(
-            itemWithResIdAndDescription(
-                "$packageName:id/new_tab_button",
-                getStringResource(R.string.resync_button_content_description),
-            ),
-        )
     }
```

Comment:
Just want to confirm, do we were expecting that https://bugzilla.mozilla.org/show_bug.cgi?id=1811976 was happening?

Reply:
This UI test assertion was confirming that the bug was happening. We don't want the "Sync" button to be displayed when the user is not signed into FxA

Reply:
Thanks for validating :)